recipe/meta.yaml

```diff
@@ -16,26 +16,41 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8,<3.12
     - pip
   run:
-    - python >=3.6
-    - tiktoken
-    - dataclasses-json
-    - langchain >=0.0.303
-    - sqlalchemy >=2.0.15
-    - numpy
-    - tenacity >=8.2.0,<9.0.0
-    - openai >=0.26.4
-    - pandas
-    - urllib3 <2
-    - fsspec >=2023.5.0
-    - typing_inspect >=0.8.0
-    - typing-extensions >=4.5.0
-    - beautifulsoup4
-    - nest-asyncio
-    - nltk
-    - tree_sitter_languages
+    - python >=3.8<=3.12
+    - SQLAlchemy ={extras = ["asyncio"], version = ">=1.4.49"}
+    - beautifulsoup4 ="^4.12.2"
+    - dataclasses-json ="*"
+    - deprecated =">=1.2.9.3"
+    - fsspec =">=2023.5.0"
+    - httpx ="*"
+    - langchain ={optional = true, version = ">=0.0.303"}
+    - nest-asyncio ="^1.5.8"
+    - nltk ="^3.8.1"
+    - numpy ="*"
+    - openai =">=1.1.0"
+    - pandas ="*"
+    - tenacity =">=8.2.0,<9.0.0"
+    - tiktoken =">=0.3.3"
+    - typing-extensions =">=4.5.0"
+    - typing-inspect =">=0.8.0"
+    - requests =">=2.31.0"
+    - asyncpg ={optional = true, version = "^0.28.0"}
+    - pgvector ={optional = true, version = "^0.1.0"}
+    - psycopg-binary ={optional = true, version = "^3.1.12"}
+    - optimum ={extras = ["onnxruntime"], optional = true, version = "^1.13.2"}
+    - sentencepiece ={optional = true, version = "^0.1.99"}
+    - transformers ={extras = ["torch"], optional = true, version = "^4.34.0"}
+    - guidance ={optional = true, version = "^0.0.64"}
+    - lm-format-enforcer ={optional = true, version = "^0.4.3"}
+    - jsonpath-ng ={optional = true, version = "^1.6.0"}
+    - rank-bm25 ={optional = true, version = "^0.2.2"}
+    - scikit-learn ={optional = true, version = "<1.3.0"}
+    - spacy ={optional = true, version = "^3.7.1"}
+    - aiostream ="^0.5.2"
```

Comment:
```suggestion
    - aiostream = ">=0.5.2,<0.6.0"
```

Reply:
sorry, I meant 

```
    - aiostream >=0.5.2,<0.6.0
```

I fixed it in https://github.com/conda-forge/llama-index-feedstock/pull/20/commits/2ded4da8d4dba0eeb8851813330e5dbf4bb7a654